fast64_internal/sm64/sm64_objects.py

```diff
@@ -1403,6 +1420,784 @@ def draw_act(self, obj, layout, value):
         layout.prop(obj, "sm64_obj_use_act" + str(value), text="")
 
 
+class AddBehavior(bpy.types.Operator):
+    bl_idname = "bone.add_behavior_script"
```

Comment:
I know it doesn´t matter functionality wise but... bone?

Reply:
It was what warps did iirc

Reply:
still, shouldn´t it be named scene.add_behavior_script instead?